cmake/Platform/Common/RuntimeDependencies_common.cmake

```diff
@@ -113,7 +113,7 @@ function(ly_get_runtime_dependencies ly_RUNTIME_DEPENDENCIES ly_TARGET)
             unset(target_locations)
             get_target_property(target_locations ${ly_TARGET} ${imported_property})
             if(target_locations)
-                list(APPEND all_runtime_dependencies ${target_locations})
+                list(APPEND all_runtime_dependencies "${target_locations}")
```

Comment:
Are you re-using the same branch?

These changes are leaking in from the Runtime Dependency update changes.

Reply:
ah, sorry, will put this in draft, apparently I branched from another branch